docs/data/material/migration/migration-v5/migration-v5.md

```diff
@@ -117,3 +159,92 @@ Both of these changes might slightly affect your layout.
 Note that the items' position doesn't change.
 We recommend adopting this new behavior and **not trying to replicate the old one**, as this is a more predictable and modern approach.
 :::
+
+## Stabilized APIs
+
+### CssVarsProvider and extendTheme
+
+The `CssVarsProvider` and `extendTheme` APIs are now stable. If you are using them in v5, you can now use them without the prefix.
+
+```diff
+- import { experimental_extendTheme as extendTheme, Experimental_CssVarsProvider as CssVarsProvider } from '@mui/material/styles';
++ import { extendTheme, CssVarsProvider } from '@mui/material/styles';
+```
+
+To learn more about using CSS theme variables, check out the [CSS theme variables page](/material-ui/customization/css-theme-variables/overview/).
+
+### Specific mode styles
+
+The `theme.applyStyles` is a new utility for creating style for a specific mode.
+It's designed to replace the use of `theme.palette.mode` when applying light or dark styles.
+
+```diff
+ const MyComponent = styled('button')(({ theme }) => ({
+   padding: '0.5rem 1rem',
+   border: '1px solid,
+-  borderColor: theme.palette.mode === 'dark' ? '#fff' : '#000',
++  borderColor: '#000',
++  ...theme.applyStyles('dark', {
++    borderColor: '#fff',
++  })
+ }))
+```
+
+Use these codemods to migrate your project to `theme.applyStyles`:
+
+```bash
+npx @mui/codemod@next v6.0.0/styled <path/to/folder-or-file>
+npx @mui/codemod@next v6.0.0/sx-prop <path/to/folder-or-file>
+npx @mui/codemod@next v6.0.0/theme-v6 <path/to/theme-file>
+```
+
+> Run `v6.0.0/theme-v6` against the file that contains the custom `styleOverrides`. Ignore this codemod if you don't have a custom theme.
+
+## Deprecations
+
+### Components and classes
+
+For more details on each component, check out the [deprecations page](/material-ui/migration/migrating-from-deprecated-apis/).
+
+### System props
+
+System props are deprecated in `Box`, `Typography`, `Link`, `Grid`, and `Stack` components.
+Move all system props into the `sx` prop by using the codemod below:
+
+```bash
+npx @mui/codemod@next v6.0.0/system-props <path/to/folder>
+```
+
+Or do it manually like the example below:
+
+```diff
+- <Button mr={2}>...</Button>
++ <Button sx={{ mr: 2 }}>...</Button>
+```
+
+### Theme component variants
+
+Theme component variants has been merged with the theme style overrides by targeting the `root` slot of the component:
```

Comment:
```suggestion
Custom component variants defined in the theme are now merged with the theme style overrides, defined within the `root` slot of the component:
```

Changing the tense here. Also, saying "custom component variants defined in theme" sounds clearer than "theme component variants", which I had to stop for a second to understand what it was.